php/async/Exchange.php

```diff
@@ -94,7 +94,7 @@ public function fetch($url, $method = 'GET', $headers = null, $body = null) {
             $this->lastRestRequestTimestamp = $this->milliseconds();
 
             try {
-                $result = React\Async\await($this->browser->request($method, $url, $headers, $body));
+                $result = React\Async\await($this->browser->request($method, $url, $headers, $body ?? ''));
```

Comment:
@gajahlemu Did you get this error when using ccxt, or do you know a realistic scenario where this could be a problem?

Reply:
@carlosmiei it only happen when use dev version of `react/http (1.x-dev)`  package

Reply:
@gajahlemu alright, I will merge it soon, thanks for your contribution